app/src/main_process/index.ts

```diff
@@ -55,6 +55,13 @@ const createWindow = (): void => {
     {
       label: 'Audapolis',
       submenu: [
+        {
```

Comment:
can you split the new window stuff into a seperate commit?

Reply:
done